packages/Babylonian-Printbugger.package/BPSwimlane.class/instance/newTitleMorphFor..st

```diff
@@ -1,14 +1,19 @@
 private
 newTitleMorphFor: aBPExample
 	
-	^ Morph new
+	| exampleLabel |
+	exampleLabel := ((aBPExample uniqueIdentifier subStrings: '>>') fold:[:aString :otherString| aString, Character cr, otherString]) asText.
```

Comment:
you say here "split the identifier at >> and concat the substrings to one" effectively just removing the ">>" from the string. the reason it's split and added as seperate morphs so you don't get one long ass line which could, with long method and class names, result in very ugly design. i'd prefer it the way it was before.
```
((aBPExample uniqueIdentifier subStrings: '>>') collect: [:aString | aString asMorph]);

Reply:
To still make use of the TextAttribute it's now shortened to 
```
(aBPExample uniqueIdentifier copyReplaceAll: '>>' with: String cr) asText
```

Reply:
Changed it back to use your suggested solution with StringMorphs